specification/applicationinsights/resource-manager/readme.md

```diff
@@ -249,6 +249,32 @@ directive:
     reason: Pre-existing error in another API (i.e. this PR did not introduce this issue).
 ```
 
+
+### Tag: package-2022-08-23
+
+These settings apply only when `--tag=package-2022-08-23` is specified on the command line.
+
+```yaml $(tag) == 'package-2022-08-23'
+input-file:
+  - Microsoft.Insights/stable/2020-02-02/components_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentAnnotations_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentApiKeys_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentContinuousExport_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentFeaturesAndPricing_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentProactiveDetection_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentWorkItemConfigs_API.json
+  - Microsoft.Insights/stable/2015-05-01/favorites_API.json
+  - Microsoft.Insights/stable/2015-05-01/webTestLocations_API.json
+  - Microsoft.Insights/stable/2022-06-15/webTests_API.json
+  - Microsoft.Insights/stable/2015-05-01/analyticsItems_API.json
+  - Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json
+  - Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json
+  - Microsoft.Insights/stable/2021-03-08/myworkbooks_API.json
+  - Microsoft.Insights/stable/2022-04-01/workbooks_API.json
+  - Microsoft.Insights/stable/2021-10-14/livetoken_API.json
+  - Microsoft.Insights/preview/2020-03-01-preview/componentLinkedStorageAccounts_API.json
+  - Microsoft.Insights/stable/2021-09-01/privateLinkScopes_API.json
```

Comment:
>2021-09-01 [](http://example.com/codeflow?start=30&length=10)

should this not be 2022-09-01 instead of 2021-09-01 as this is a new apiversion? also, is this going straight into stable and not preview right?

Reply:
This is a stable version that has existed in the manifest and has been in use for a while (since 2021-09-01...), they just never did a Swagger for it. The reason I'm doing a swagger is actually because a customer noticed there's no docs and refused to use it in their project :) So I would like to keep the version name the same as the api-version here, I think it makes sense. For the version of the package (the tag) I will use a current date (the date this PR was submitted).